ui/src/components/SubPlayPage.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import DrawingCanvas from "./DrawingCanvas";
+
+const SubPlayPage = ({number_truth = "0"}) => {   
+    const [canvasURI, setCanvasURI] = useState("")  
+    const [number_result, SetNumber] = useState(0);
+    const [score_result, SetScore] = useState(0.99);
```

Comment:
I think it's better to have this const [scoreResult, SetScoreResult] = useState(0);
just a consistent format among the whole project

Reply:
I can add those changes after the merge